modules/xmpp/JingleSessionPC.js

```diff
@@ -196,7 +196,7 @@ export default class JingleSessionPC extends JingleSession {
         this._gatheringReported = false;
 
         this.lasticecandidate = false;
-        this.closed = false;
+        this._closed = false;
```

Comment:
is there a reason why we have this `closed` member when we already have `this.state`?

Reply:
by that i mean, why doesn't whatever sets `closed` to true just change the state instead? why 2 things?

Reply:
Yeah, so the thing is that `state` is updated after things go through a chain of callbacks here and there and when finally `onTerminated` will be called. I supposed it only even gets called when jingle 'session-terminate' is received. So this `state` is more like a Jingle state, but maybe I could use it instead of `_closed`. Currently the `_closed` flag guards against calling close more than once. I was just a bit scared to touch it to be honest :P

Reply:
>by that i mean, why doesn't whatever sets closed to true just change the state instead? why 2 things?

Right, I was afraid to not break some callbacks, but I'll give it a try and we'll see.

Reply:
gotcha, i see.  i know it was already there, but since we're now relying on it more i was wondering because it seems more complex to have 2 things modeling this state.